packages/conform-react/integrations.ts

```diff
@@ -294,10 +294,12 @@ export function useInputValue<
 	return [value, setValue] as const;
 }
 
-export function useControl<
+export function unstable_useControl<
 	Value extends string | string[] | Array<string | undefined>,
 >(meta: { key?: Key | null | undefined; initialValue?: Value | undefined }) {
+	// eslint-disable-next-line react-hooks/rules-of-hooks
 	const [value, setValue] = useInputValue(meta);
+	// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Err.. why?

Reply:
since the name of the function doesn't start with `use` it complains, I'm not sure what remix team does with thier unstable APIs to get around this but I could look into it. I'm guessing they might have lint rules turned off for things marked as `unstable_`